sdk/python/kfp/v2/v2_namespace_alias_test.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2022 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=import-outside-toplevel,unused-import,import-error
+import json
+import os
+import sys
+import tempfile
+import unittest
+
+
+class V2NamespaceAliasTest(unittest.TestCase):
+    """Test that imports of both modules and objects are aliased
+    (e.g. all import path variants work).
+    """
+
+    @classmethod
+    def setUp(cls):
+        # need to unload the kfp.v2 module before each test to ensure that the Deprecation warning is raised each time
+        keys = {k for k, _ in sys.modules.items() if k.startswith("kfp")}
+        for key in keys:
+            del sys.modules[key]
+
+    def test_import_namespace(self):  # pylint: disable=no-self-use
+        with self.assertWarns(DeprecationWarning):
```

Comment:
There might be other deprecation warnings. Consider using `assertWarnsRegex`, which allows you to (partially) match the warning message. 

e.g.: https://github.com/kubeflow/pipelines/blob/b693909a0879d478c4cb8711932b06c956629ae0/sdk/python/kfp/deprecated/components_tests/test_components.py#L1106-L1110